### ISSUE TITLE
Highlight Pinned articles

### ISSUE THREAD

**Issue by madzadev** [forem/forem#8819]
**Is your feature request related to a problem? Please describe.**

I really like the 'pin article' option in user's profile to highlight particular posts.
However, as the user is intended to get an extra attention to them, they should stand out more compared to regular posts.

**Describe the solution you'd like**

I would suggest adding some styling to pinned articles.
You could add a new class like 'crayons-story__pinned' for those articles that get pinned and add like 'background-color: ghostwhite' to highlight them:

![AYFAjsCrIc](https://user-images.githubusercontent.com/42511917/85250764-30fd7b00-b460-11ea-95ee-917ac114f641.png)

**Comment by citizen428** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by nickytonline** [comment]
cc: @ludwiczakpawel @lisasy

**Comment by ludwiczakpawel** [comment]
So the pinned articles are already in designated area on top of your profile... So these are highlighted in a way, right? Do we need to highlight them more?

**Comment by madarsbiss** [comment]
@ludwiczakpawel thanks for looking into this feature request! Yeah, I don't think that just positioning them on top is enough.

When I open some profile that has some pinned articles, it's hard to immediately distinguish where the pinned articles end and where the latest articles start (because visually they look the same). An extra darker shade of background color would be nice, so it's possible **to distinguish right away** without paying attention where the pinned container ends.

**Comment by ludwiczakpawel** [comment]
@madarsbiss If you don't mind lets put it on hold for now. We will soon be working on redesigning/updating the user profile view and I think this would be good moment to revise pinned articled as well. As long as it's not a bug (and clearly it's not one) I think it's safe to punt for now. Is that ok with you?

**Comment by ludwiczakpawel** [comment]
@madzadev this view has changed significantly since this issue has been created and I promised we will circle back. Can you give it another look and let me know what do you think?
The "pinned" area has been made a bit more "loud" visually but I'm keen to know your thoughts.
